app/services/data_migrations/backfill_selected_boolean.rb

```diff
@@ -0,0 +1,12 @@
+module DataMigrations
+  class BackfillSelectedBoolean
+    TIMESTAMP = 20210517161321
+    MANUAL_RUN = false
+
+    def change
+      ApplicationReference.feedback_provided.find_in_batches(batch_size: 100) do |batch|
```

Comment:
```suggestion
      ApplicationReference.feedback_provided.find_in_batches  do |batch|
```

default batch size of 1000 should be fine here I reckon